From 46f5006361350be7eb076c8c968e1cdfc4eaad26 Mon Sep 17 00:00:00 2001
From: Mike FABIAN <mfabian@redhat.com>
Date: Wed, 20 May 2015 11:16:30 +0200
Subject: [PATCH 4/5] Add transliteration rules for da, nb, nn, and sv locales.

for localedata/Changelog

    [BZ #89]
    * locales/da_DK add more transliteration rules
    * locales/nb_NO add transliteration rules
    * locales/sv_SE add transliteration rules
---
 localedata/locales/da_DK | 21 ++++++++++++++++++---
 localedata/locales/nb_NO | 22 ++++++++++++++++++++++
 localedata/locales/sv_SE | 22 ++++++++++++++++++++++
 3 files changed, 62 insertions(+), 3 deletions(-)

diff --git a/localedata/locales/da_DK b/localedata/locales/da_DK
index a3c5561..3248e3c 100644
--- a/localedata/locales/da_DK
+++ b/localedata/locales/da_DK
@@ -136,11 +136,26 @@ translit_start
 
 include "translit_combining";""
 
-% Danish.
-% LATIN CAPITAL LETTER A WITH RING ABOVE.
+% LATIN CAPITAL LETTER A WITH DIAERESIS -> "AE"
+<U00C4> "<U0041><U0308>";"<U0041><U0045>"
+% LATIN CAPITAL LETTER A WITH RING ABOVE -> "AA"
 <U00C5> "<U0041><U030A>";"<U0041><U0041>"
-% LATIN SMALL LETTER A WITH RING ABOVE.
+% LATIN CAPITAL LETTER AE -> "AE"
+<U00C6> "<U0041><U0045>"
+% LATIN CAPITAL LETTER O WITH DIAERESIS -> "OE"
+<U00D6> "<U004F><U0308>";"<U004F><U0045>"
+% LATIN CAPITAL LETTER O WITH STROKE -> "OE"
+<U00D8> "<U004F><U0338>";"<U004F><U0045>"
+% LATIN SMALL LETTER A WITH DIAERESIS -> "ae"
+<U00E4> "<U0061><U0308>";"<U0061><U0065>"
+% LATIN SMALL LETTER A WITH RING ABOVE -> "aa"
 <U00E5> "<U0061><U030A>";"<U0061><U0061>"
+% LATIN SMALL LETTER AE -> "ae"
+<U00E6> "<U0061><U0065>"
+% LATIN SMALL LETTER O WITH DIAERESIS -> "oe"
+<U00F6> "<U006F><U0308>";"<U006F><U0065>"
+% LATIN SMALL LETTER O WITH STROKE -> "oe"
+<U00F8> "<U006F><U0338>";"<U006F><U0065>"
 
 translit_end
 
diff --git a/localedata/locales/nb_NO b/localedata/locales/nb_NO
index 513d50c..332092a 100644
--- a/localedata/locales/nb_NO
+++ b/localedata/locales/nb_NO
@@ -127,6 +127,28 @@ copy "i18n"
 
 translit_start
 include  "translit_combining";""
+
+% LATIN CAPITAL LETTER A WITH DIAERESIS -> "AE"
+<U00C4> "<U0041><U0308>";"<U0041><U0045>"
+% LATIN CAPITAL LETTER A WITH RING ABOVE -> "AA"
+<U00C5> "<U0041><U030A>";"<U0041><U0041>"
+% LATIN CAPITAL LETTER AE -> "AE"
+<U00C6> "<U0041><U0045>"
+% LATIN CAPITAL LETTER O WITH DIAERESIS -> "OE"
+<U00D6> "<U004F><U0308>";"<U004F><U0045>"
+% LATIN CAPITAL LETTER O WITH STROKE -> "OE"
+<U00D8> "<U004F><U0338>";"<U004F><U0045>"
+% LATIN SMALL LETTER A WITH DIAERESIS -> "ae"
+<U00E4> "<U0061><U0308>";"<U0061><U0065>"
+% LATIN SMALL LETTER A WITH RING ABOVE -> "aa"
+<U00E5> "<U0061><U030A>";"<U0061><U0061>"
+% LATIN SMALL LETTER AE -> "ae"
+<U00E6> "<U0061><U0065>"
+% LATIN SMALL LETTER O WITH DIAERESIS -> "oe"
+<U00F6> "<U006F><U0308>";"<U006F><U0065>"
+% LATIN SMALL LETTER O WITH STROKE -> "oe"
+<U00F8> "<U006F><U0338>";"<U006F><U0065>"
+
 translit_end
 END LC_CTYPE
 
diff --git a/localedata/locales/sv_SE b/localedata/locales/sv_SE
index ecf7858..92358b9 100644
--- a/localedata/locales/sv_SE
+++ b/localedata/locales/sv_SE
@@ -112,6 +112,28 @@ copy "i18n"
 
 translit_start
 include  "translit_combining";""
+
+% LATIN CAPITAL LETTER A WITH DIAERESIS -> "AE"
+<U00C4> "<U0041><U0308>";"<U0041><U0045>"
+% LATIN CAPITAL LETTER A WITH RING ABOVE -> "AA"
+<U00C5> "<U0041><U030A>";"<U0041><U0041>"
+% LATIN CAPITAL LETTER AE -> "AE"
+<U00C6> "<U0041><U0045>"
+% LATIN CAPITAL LETTER O WITH DIAERESIS -> "OE"
+<U00D6> "<U004F><U0308>";"<U004F><U0045>"
+% LATIN CAPITAL LETTER O WITH STROKE -> "OE"
+<U00D8> "<U004F><U0338>";"<U004F><U0045>"
+% LATIN SMALL LETTER A WITH DIAERESIS -> "ae"
+<U00E4> "<U0061><U0308>";"<U0061><U0065>"
+% LATIN SMALL LETTER A WITH RING ABOVE -> "aa"
+<U00E5> "<U0061><U030A>";"<U0061><U0061>"
+% LATIN SMALL LETTER AE -> "ae"
+<U00E6> "<U0061><U0065>"
+% LATIN SMALL LETTER O WITH DIAERESIS -> "oe"
+<U00F6> "<U006F><U0308>";"<U006F><U0065>"
+% LATIN SMALL LETTER O WITH STROKE -> "oe"
+<U00F8> "<U006F><U0338>";"<U006F><U0065>"
+
 translit_end
 END LC_CTYPE
 
-- 
2.4.3


